test/export/test_converter.py

```diff
@@ -25,7 +25,7 @@ def setUp(self):
         init_torchbind_implementations()
 
         @torch._library.register_fake_class("_TorchScriptTesting::_TensorQueue")
-        class FakeTensorQueue:
+        class FakeTensorQueue:  # noqa: F841
```

Comment:
Ho it doesn't detect that the decorator above is having a side effect on this function?

Reply:
You're asking for an awful lot from `ruff` here. 😁 

I think it's only a minority of decorators that actually register their arguments globally, and how is it supposed to tell the difference?

I decided however to rename the class to `class _FakeTensorQueue:` which should stop the error, avoid the `#noqa`, and make it clear that this symbol isn't actually used.